packages/docs-app/src/examples/datetime-examples/dateRangeInputExample.tsx

```diff
@@ -45,12 +45,13 @@ export class DateRangeInputExample extends BaseExample<IDateRangeInputExampleSta
     private toggleSingleDay = handleBooleanChange(allowSingleDayRange => this.setState({ allowSingleDayRange }));
 
     protected renderExample() {
-        return <DateRangeInput {...this.state} />;
+        const { formatKey, ...otherState } = this.state;
+        return <DateRangeInput format={FORMATS[formatKey]} {...otherState} />;
```

Comment:
Same note RE: `spreadableState`. We use `spreadableProps` elsewhere in the codebase, FWIW.